tailwind.config.js

```diff
@@ -200,10 +200,15 @@ module.exports = {
           '50%': { opacity: '0' },
           '100%': { opacity: '1' },
         },
+        wiggle: {
+          '0%, 100%': { transform: 'rotate(-25deg)' },
+          '50%': { transform: 'rotate(25deg)' },
+        },
       },
       animation: {
         'infinite-slide': 'infinite-slide 30s linear infinite',
         'blink-fast': 'blink 1s step-start infinite',
+        'spin-slow': 'spin 12s linear infinite',
```

Comment:
@ryan-gang I think this was supposed to be 16s - can you confirm with Louis once please?